package.json

```diff
@@ -313,6 +313,7 @@
     "md5-file": "4.0.0",
     "mixedindentlint": "1.2.0",
     "mockdate": "2.0.2",
+    "moment-timezone-data-webpack-plugin": "1.0.1",
```

Comment:
Shouldn't this be a `dependency` rather that `devDependency`? We have this weird requirement for production Docker builds. I'm suprised that, e.g., calypso.live build is not failing here.

Maybe `lerna bootstrap` changed the game and now always installs `devDependencies`, too?

Reply:
Hmm, I don't know if there are any special requirements around dependencies in Calypso. I merely added this as a `devDependency` as I would in any other project, since it only is needed at build time, and everything seemed to work across my local environment and the various CI tools.

Do let me know if I should change it, though!

Reply:
> We have this weird requirement for production Docker builds. 

We dropped that need a while ago, I think as part of the move to lerna, but perhaps before that even. We could (should?) move all the calypso dependencies to devDependencies now?